src/main/java/ua/goit/timonov/enterprise/module_6_2/controllers/DishController.java

```diff
@@ -0,0 +1,81 @@
+package ua.goit.timonov.enterprise.module_6_2.controllers;
+
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import ua.goit.timonov.enterprise.module_6_2.dao.DishDAO;
+import ua.goit.timonov.enterprise.module_6_2.model.Dish;
+
+import java.util.List;
+
+/**
+ * Controller for DishDAO
+ */
+public class DishController {
+
+    private DishDAO dishDAO;
+
+    public void setDishDAO(DishDAO dishDAO) {
+        this.dishDAO = dishDAO;
+    }
+
+    /**
+     * finds list of all dishes in DB
+     * @return          list of dishes
+     * throws               EmptyResultDataAccessException, DataAccessException
+     */
+    @Transactional(propagation = Propagation.REQUIRED)
```

Comment:
According to http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html#propagation--
Propagation value is Propagation.REQUIRED by default. I suppose you should omit explicit Propagation.REQUIRED declaration